Lombiq.HelpfulLibraries.OrchardCore/TagHelpers/EditorFieldSetTagHelper.cs

```diff
@@ -75,31 +82,31 @@ public override Task ProcessAsync(TagHelperContext context, TagHelperOutput outp
         return Task.CompletedTask;
     }
 
-    private void AppendInputAndLabel(TagHelperOutput output, bool isRequired)
+    private void AppendInputAndLabel(TagHelperOutput output, bool isRequired, ModelExpression forExpression, string labelHtml)
     {
         var label = _htmlGenerator.GenerateLabel(
             ViewContext,
-            For.ModelExplorer,
-            For.Name,
-            Label.Html().Trim() + (isRequired ? " *" : string.Empty),
+            forExpression.ModelExplorer,
+            forExpression.Name,
+            labelHtml + (isRequired ? " *" : string.Empty),
             htmlAttributes: null);
 
         var attributes = new Dictionary<string, object>();
         AddBoolAttribute(attributes, IsReadOnly, "readonly");
         AddBoolAttribute(attributes, isRequired, "required");
 
-        if (InputType.EqualsOrdinalIgnoreCase("checkbox"))
+        if (StringExtensions.EqualsOrdinalIgnoreCase(InputType, (string?)"checkbox"))
```

Comment:
Why do you think `InputType!.EqualsOrdinalIgnoreCase((string?)"checkbox")` isn't finding our extension here (with the `string, string?` signature)?

Reply:
Yeah, I understand. I'm asking why do you think method resolution doesn't find ours when to me it looks like it should.